### DISCUSSION TITLE
代码格式化问题

### DISCUSSION THREAD

**Discussion by shilin8805** [opensumi/core#1935] [Q&A] [answered]
咨询下，opensumi内置代码格式化的功能吗？还是需要安装第三方插件支持？
我看支付宝小程序开发者工具里面右键菜单内有代码格式化的一些命令，OpenSumi默认的没有，这个是需要自己实现吗？
支付宝的：
<img width="619" alt="image" src="https://user-images.githubusercontent.com/3340210/200458756-2ffc2d5c-4018-4614-af43-8647d723ba12.png">
OpenSumi默认的：
<img width="411" alt="image" src="https://user-images.githubusercontent.com/3340210/200458961-d69a5d1c-c42c-4bf5-970f-39892b988a06.png">

**Comment by erha19** [comment, marked as answer]
@shilin8805 需要你自行安装格式化插件才能激活相应菜单，如 Vue 类型常用的格式化插件 `febean.vue-format`
